setup_iOS_build_environment.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
+# Set the release version to download, default to current supported if not provided
+RELEASE_VERSION=${1:-6.5.2}
```

Comment:
should this be 6.5.1?

Reply:
This should be 6.5.2 actually; we've shifted the target I think and I need to rename the branch.